src/components/DpButton/DpButton.vue

```diff
@@ -50,7 +85,25 @@ export default {
     },
 
     /**
-     * The text content of the button element. Can be omitted if the text content is passed via the default slot.
+     * Icon that will be placed before button text.
+     */
+    icon: {
+      required: false,
+      type: String,
+      default: ''
+    },
+
+    /**
+     * Icon that will be placed after button text.
+     */
+    iconAfter: {
```

Comment:
I thought this would be a boolean, or is there a case in which we have an icon both before and after the button text?

Reply:
even if it is not a relevant use case to support two icons at the same time, we would need to define both the `icon` and `iconAfter` prop to render an icon after the text. So, in this case i'd opt for a little bit of magic to have a simpler component api.